### DISCUSSION TITLE
Can't deploy my apps anymore

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [vercel/community#6403] [Bug Report]
### Page to Investigate

https://vercel.com/juliomuhlbauer/ruralize-lands/5BNPEQG6brLxMZPzP21yM5D4aUf4

### Steps to Reproduce

Related: https://github.com/vercel/vercel/issues/11102

![image](https://github.com/vercel/community/assets/53458125/c297d6f9-e920-4f4c-8ec7-c632dd5ce7d8)

**Comment by versecafe** [comment]
@juliomuhlbauer https://vercel.com/juliomuhlbauer/ruralize-lands/5BNPEQG6brLxMZPzP21yM5D4aUf4 is not something others can view could you share a github link to your repo, but also the error appears to be with CommonJS vs ESM imports, you may want to try setting esModuleInterop to true in your tsconfig, https://www.typescriptlang.org/tsconfig#esModuleInterop

**Reply by juliomuhlbauer** [reply]
Now it is working, I don't know why. Also esModuleInterop was set to true
